webui/react/src/react-app-env.d.ts

```diff
@@ -13,5 +13,8 @@ export declare global {
   interface Window {
     analytics: any;
     dev: any;
+    dai: {
+      CLUSTER_ADDRESS: string;
+    };
```

Comment:
**nit:** keep `clusterAddress` inside `dev`?

Reply:
old commit :)

Reply:
this doesn't apply anymore right?